examples/express/README.md

```diff
@@ -0,0 +1,87 @@
+# Express.js example
+
+0. Installation
+
+    Navigate to the example subdirectory, for example:
```

Comment:
Suggestion: If you are comfortable with it, you could use: Navigate to the example subdirectory: